torchrl/objectives/value/advantages.py

```diff
@@ -365,7 +365,7 @@ def is_stateless(self):
         return self.value_network._is_stateless
 
     def _next_value(self, tensordict, target_params, kwargs):
-        step_td = step_mdp(tensordict)
+        step_td = step_mdp(tensordict, keep_other=False)
```

Comment:
We need this because in case "next" does not contain the new inputs for the value network, the old ones will be silently brought forward and no error would be thrown. If we set it to False, no state from the previous step is brought forward and we gin perf